### PR TITLE
Refactor CSV file used to import pegasus data to be shared with dashb…

### PR DESCRIPTION
**Pull request by pablo-code-org**
- This is breaking the existing CSV file into their own file based on the library invoked. 
- This makes the split between pegasus and dashboard more explicit
- Also we already had a CSV.rb on the shared library which also might need renaming but not included in this diff

## Testing story
` bundle exec rake seed:migrate_dashboard --trace` executes correctly.

`bundle exec rake seed:sync --trace` (invoking pegasus code) works*

`bundle exec rake seed:reset --trace` (forced csv to sql pegasus) works

* works partially since I don't have the google sheet credentials. But it invokes the library correctly. 
I also verified pegasus/data/static_models.rb works (:migrate invokes this method)

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: lib/cdo/data/google_sheet_to_csv.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      end
+    end
+
+    File.utime(File.atime(@csv_path), @file.mtime, @csv_path)
+
+    ChatClient.log "Downloaded <b>#{@gsheet_path}</b> (<b>#{File.size(@csv_path)}</b> btyes) from Google Drive."
+
+    return @csv_path
+  end
+end
```
Comment (davidsbailey) [inline lib/cdo/data/google_sheet_to_csv.rb:77]: the new filenames look good to me!

**Review by davidsbailey — COMMENTED** [review]
Hey Pablo, code changes look good, but a few notes on testing:

1. the way drone works, it won't run on PRs that do not have staging as the base branch, so please make sure you get a green drone run before merging. I would expect drone to notice if a page that's backed by pegasus gsheet tables was broken.
2. `bundle exec rake seed:migrate_dashboard --trace` does not match any rake task I am aware of. `cd dashboard; rake seed:import_pegasus_data` would be a good choice for validating that this is working from the dashboard side.

once you've done that validation, and merged the other PR, this PR looks good to merge into staging.

**Comment by pablo-code-org** [comment]
Sorry I meant `seed:import_pegasus_data` on the test plan. 

I didn't know drone didn't run on this :(

**Comment by davidsbailey** [comment]
> I didn't know drone didn't run on this :(

once you merge the other PR, you can merge latest staging into this PR, and that will make drone run.

for next time, if you are going to chain PRs together, one idea is to open the 2nd PR against staging, and then wait until you have a green drone run before changing the base branch.

**Comment by pablo-code-org** [comment]
I am reverting some files that were added while I was trying to force drone to run

**Comment by cat5inthecradle** [comment]
> I am reverting some files that were added while I was trying to force drone to run

Pro git tip!

`git commit --allow-empty -m "My empty commit with a message"`

**Comment by pablo-code-org** [comment]
OMG, Very useful! I don't know how it didn't cross my mind to send an empty commit  Thanks @cat5inthecradle

**Review by davidsbailey — APPROVED** [review]
this cleaned up very nicely, thank you for doing this!

**Pull request merged**